examples/cross-contract-calls/README.md

```diff
@@ -1,5 +1,58 @@
-# Cross contract
+# Cross Contract Call
 
-Example of using cross-contract functions with promises.
+Example implementation of a cross-contract call using [near-sdk-rs].
 
-TBD
+[near-sdk-rs]: https://github.com/near/near-sdk-rs
+
+NOTES:
+
+- This example demonstrates how to call another contract from within a contract.
+- Ensure that the called contract is deployed and accessible.
+
+## Project Structure
+
+- `.cargo`: Configuration files for Cargo, the Rust package manager.
+- `high-level/src`: Source code for the high-level cross-contract call example.
+- `low-level/src`: Source code for the low-level cross-contract call example.
+- `res`: Compiled contract binaries.
+- `tests`: Integration tests for the cross-contract call examples.
+
+## Building
+
+To build run:
+
+```bash
+./build.sh
+```
+
+## Testing
+
+To test run:
+
+```bash
+cargo test --package cross-contract-call -- --nocapture
+```
+
+### Usage
+
+```markdown
+## Usage
+
+1. Deploy the contracts:
+
+   - Deploy the high-level contract.
+   - Deploy the low-level contract.
+
+2. Call the high-level contract to initiate a cross-contract call to the low-level contract.
```

Comment:
Please, document how to do that with `cargo near deploy`:

1. Prerequisite: [install `cargo-near`](https://github.com/near/cargo-near)
2. (optional) Create a new testnet account: `cargo near create-dev-account`
3. Build and deploy the contract: `cargo near deploy` + add a screenshot here
4. Interact with the contract from [CLI](https://near.cli.rs): `near contract call-function ...` + add a screenshot here

Reply:
Just for the reference, I have tested it here:

<img width="1151" alt="image" src="https://github.com/user-attachments/assets/860e2cf5-39b2-4441-b1b5-4127088af908">

P.S. There is a small bug in near.cli.rs, so you have to change the `pub fn factorial(&self` into `pub fn factorial(&mut self`

Reply:
Got it @frol, will do as soon as possible!

Reply:
Just made the changes